src/test/functional/features/sole-hub-page.feature

```diff
@@ -100,6 +100,13 @@ Feature: Sole hub page
     And the page should include "The hearing will take place at Birmingham Civil and Family Justice Centre on 29 September 2013 at 3:30PM."
     Then the page should include "You can view and download your 'certificate of entitlement for a conditional order'."
 
+  Scenario: Hub Awaiting clarification state
+    Given I set the case state to "AwaitingClarification"
+    And a superuser updates "coRefusalClarificationAdditionalInfo" with "Refusal reason test"
+    Given I click "Sign out"
+    And I login with applicant "1"
+    Then the page should include "Refusal reason test"
```

Comment:
I'd add a check here too that 'What you need to do now' shows up instead of 'latest update'